rfcs/text/0013-multiple-files.md

```diff
@@ -0,0 +1,220 @@
+# 0013: Multiple files in an event
```

Comment:
```suggestion
# 0000: Multiple files in an event
```

It may remain as `0013`, but just a note the RFC number will be officially allocated when this PR is merged.